crates/cheatcodes/src/evm.rs

```diff
@@ -224,6 +224,24 @@ impl Cheatcode for resumeGasMeteringCall {
     }
 }
 
+impl Cheatcode for lastCallGasCall {
+    fn apply(&self, state: &mut Cheatcodes) -> Result {
+        let Self {} = self;
+        ensure!(
+            state.config.isolate,
+            "`lastCallGas` is only available in isolated mode (`--isolate`)"
+        );
+        ensure!(state.last_call_gas.is_some(), "`lastCallGas` is only available after a call");
+
+        Ok(state
+            .last_call_gas
+            .as_ref()
+            // This should never happen, but we need to handle it to satisfy the compiler.
```

Comment:
I'd write a bit on why this can't happen e.g we've already ensured that `last_call_gas.is_some()` above

Reply:
Added a clarifying note